lib/drivers/docs/medtronicCGM.md

```diff
@@ -0,0 +1,98 @@
+## Medtronic CGM
+
+(Key:
+
+ - `[x]` available in data protocol/documented in spec and implemented
+ - `[-]` available in data protocol/documented in spec but *not* yet implemented
+ - `[?]` unknown whether available in data protocol/documented in spec; *not* yet implemented
+ - `*[ ]` TODO: needs implementation!
+ - `[ ]` unavailable in data protocol and/or not documented in spec and not yet implemented)
+
+### Required if Present
+
+#### CBG
+
+  - `[x]` cbg values
+  - `[?]` units of cbg values (read from device, not hard-coded)
+  - `[ ]` out-of-range values (LO or HI)
```

Comment:
Oh, that applies to HI CGM readings as well? I think I misunderstood that discussion because I thought you guys were just talking about smbg readings.